src/test/java/com/google/showcase/v1beta1/NumericEnumsIT.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.showcase.v1beta1;
+
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.gax.core.NoCredentialsProvider;
+import com.google.api.gax.rpc.InvalidArgumentException;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class NumericEnumsIT {
+
+  private ComplianceClient client;
+
+  @Before
+  public void createClient() throws GeneralSecurityException, IOException {
+    ComplianceSettings complianceSettings =
+        ComplianceSettings.newHttpJsonBuilder()
+            .setCredentialsProvider(NoCredentialsProvider.create())
+            .setTransportChannelProvider(
+                ComplianceSettings.defaultHttpJsonTransportProviderBuilder()
+                    .setHttpTransport(
+                        new NetHttpTransport.Builder().doNotValidateCertificate().build())
+                    .setEndpoint("http://localhost:7469")
+                    .build())
+            .build();
+    client = ComplianceClient.create(complianceSettings);
+  }
+
+  @After
+  public void destroyClient() {
+    client.close();
+  }
+
+  // See
+  // https://github.com/googleapis/gapic-showcase/blob/v0.25.0/util/genrest/resttools/systemparam.go#L37-L46
+  @Test(expected = InvalidArgumentException.class)
```

Comment:
Usually we use `assertThrows` instead because it's clearer where exactly is the exception thrown. But I know this test case is temporary as well because the showcase server issue.

Reply:
GTK. I've made some adjustments to use the recommended `assertThrows`